crates/re_viewer/src/app_state.rs

```diff
@@ -146,12 +147,40 @@ impl AppState {
         egui::CentralPanel::default()
             .frame(central_panel_frame)
             .show_inside(ui, |ui| {
-                blueprint_panel_ui(
-                    &mut viewport.blueprint,
-                    &mut ctx,
+                let left_panel = egui::SidePanel::left("blueprint_panel")
+                    .resizable(true)
+                    .frame(egui::Frame {
+                        fill: ui.visuals().panel_fill,
+                        ..Default::default()
+                    })
+                    .min_width(120.0)
+                    .default_width((0.35 * ui.ctx().screen_rect().width()).min(200.0).round());
+
+                left_panel.show_animated_inside(
                     ui,
-                    &spaces_info,
                     app_blueprint.blueprint_panel_expanded,
+                    |ui: &mut egui::Ui| {
+                        // Set the clip rectangle to the panel for the benefit of nested, "full span" widgets like
+                        // large collapsing headers. Here, no need to extend `ui.max_rect()` as the enclosing frame
+                        // doesn't have inner margins.
+                        ui.set_clip_rect(ui.max_rect());
+
+                        // TODO(ab): this might be promoted higher in the hierarchy once list item are
+                        //  used in the blueprint panel section
```

Comment:
```suggestion
                        // used in the blueprint panel section.
```

Reply:
hehe, the reason for that is:

<img width="614" alt="image" src="https://github.com/rerun-io/rerun/assets/49431240/29ec9477-5c88-4299-9d5f-ee915de600b3">

<img width="607" alt="image" src="https://github.com/rerun-io/rerun/assets/49431240/837c5df7-ab7a-416b-8a45-c32c7c3a0e62">

always, who cares 🤷🏼